content/learn/welcome/introduction.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Introduction"
+weight = 110
++++
+
+> The first thing you gotta discard, whenever you're learning about something, is your ego. - Luke Smith
+
+These are the learning resources for YAGPDB's powerful custom command system. This is your playground!
+
+We are going to teach you how to read code as well as how to write it. After all, how can you **write** if you cannot
+**read**? Learn to read, learn to solve problems, and learn to enjoy the process. The community is there to help, as
+long as you put in some effort yourself.
+
+To understand the custom commands interface you will be interacting with on the control panel, please read the
+documentation on [custom commands](/docs/custom-commands/commands).
```

Comment:
"These guides assume that you are already familiar with the custom command UI, so, for instance, you know how to add and edit commands through the control panel. Consult the [documentation on custom commands] if unsure."